docs/program/apis/robot.md

```diff
@@ -0,0 +1,410 @@
+---
+title: "Manage Robots with Viam's Client SDKs"
+linkTitle: "Robot Management"
+weight: 20
+type: "docs"
+description: "How to use the Robot API to monitor and manage your robots."
+tags: ["robot state", "sdk", "apis", "robot api"]
+---
+
+The *robot API* is the designated interface for a robot, the root of all robotic parts.
+To interact with the robot API with Viam's SDKs, instantiate a `RobotClient` ([gRPC](https://grpc.io/) client) and use that class for all interactions.
+
+The robot API supports the following methods:
+
+{{< readfile "/static/include/services/apis/robot.md" >}}
+
+<!-- This isn't actually all of the methods; just the ones that were already in the APIs doc. -->
```

Comment:
Do you want to keep this commented line in?

Reply:
I meant to for our reference, in case some day we search the docs for comments and fill in missing info. But we don't have to I guess?